docs/tools/ngrok.md

```diff
@@ -67,3 +67,46 @@ Option | Variable | Purpose
 `--region=us` | NGROK_REGION | Location of the ngrok tunnel server; can be `us` (United States, default), `eu` (Europe), `ap` (Asia/Pacific) or `au` (Australia)
 `--debug` | NGROK_DEBUG | To debug the connection and see a more detailed log set this to 1
 `--host-header=test.example.docksal` | NGROK_HEADER | Set Host Header. Set to the domain that should be passed through to the host.
+
+## Configuration File
+
+If ngrok is being used often with the same settings to help speed the process up a configuration file can be used. The configuration file should be added to the project in `.docksal/etc/ngrok/ngrok.yml`. For more details on the configuration file view the [Ngrok Docs](https://ngrok.com/docs#config).
+
+Example `ngrok.yml`
+
+```
+# Ngrok example configuration file.
+# To use, rename the items in CAPS to your docksal host, and rename this file to ngrok.local.yml.
```

Comment:
You previously said ngrok.yml, and now this says ngrok.local.yml. It could use some clarity.